src/ui/server/controllers/insurance/your-buyer/alternative-currency/index.test.ts

```diff
@@ -197,7 +202,7 @@ describe('controllers/insurance/your-buyer/alternative-currency', () => {
       it('should call api.keystone.APIM.getCurrencies', async () => {
         await get(req, res);
 
-        expect(getCurrenciesSpy).toHaveBeenCalledTimes(1);
+        expect(getCurrenciesSpy).toHaveBeenCalledTimes(2);
```

Comment:
Why is this being called twice?

Reply:
once for supported currencies and once for all currencies